client/src/components/PitchDoc/SubmitPitchModal.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement } from 'react';
+import { Modal, Button, Form, Image } from 'semantic-ui-react';
+
+// import { currentTeamsButtons } from '../../utils/constants';
+import '../../css/pitchDoc/SubmitPitchModal.css';
+import GoogleDriveScreenshot from '../../assets/GoogleDriveScreenshot.png';
+// import WizardSelectButton from '../WizardSelectButton/WizardSelectButton';
+
+// const defaultOnClick = (): void => void 0;
+
+function SubmitPitchModal(): ReactElement {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Modal
+      onClose={() => setOpen(false)}
+      onOpen={() => setOpen(true)}
+      open={open}
+      trigger={
+        <Button className="submit-pitch-button"> Submit a Pitch </Button>
+      }
+      closeIcon
+    >
+      <Modal.Actions>
+        <Form>
+          <Form.Field>
+            Pitch Title
+            <label>
+              <input />
+            </label>
+          </Form.Field>
+          <Form.Field>
+            Summarize Pitch in 1-2 Sentences (300 character limit)
+            <label>
+              <input />
+            </label>
+          </Form.Field>
+
+          <div className="suggested-text">Suggested Roles Needed:</div>
+
+          <div className="gdrive-teams-container">
+            <Image src={GoogleDriveScreenshot}></Image>
+
+            <div className="teams-section">
+              <Form.Group widths="equal">
+                {/* {Object.keys(currentTeamsButtons).map((team: string) => (
```

Comment:
Remove anything unused here so we don't need to worry about it later. If it needs to stay add comments explaining why it needs to be in there